tests/test_update_manager_no_mocks.py

```
@@ -0,0 +1,444 @@
+
+        # Test real cleanup logic
+        await manager._cleanup_old_backups("usda")
+
+        # Should keep only max_rollback_versions files
+        usda_files = list(temp_dir.glob("usda_backup_*.json"))
+        assert len(usda_files) <= 5  # max_rollback_versions=5
+
+    def test_scheduled_update_real_logic(self):
+        """Test run_scheduled_update with real logic."""
+        # Test with real manager instance
+        manager = DatabaseUpdateManager(cache_dir=Path("/tmp"), update_interval_hours=24)
```

Ruff (0.14.1)

Probable insecure usage of temporary file or directory: "/tmp"

(S108)